Add Windows installation (#161)
ignitionrobotics/ign-gui: tutorials/01_install.md

Create if necessary, and activate a Conda environment:

```
conda create -n ign-ws
conda activate ign-ws
```

Install dependencies:
```
conda install qt --channel conda-forge
```

Install Ignition dependencies, replacing `<#>` with the desired versions:

```
conda install libignition-cmake<#> libignition-common<#> libignition-math<#> libignition-transport<#> libignition-msgs<#> libignition-plugin<#> libignition-tools<#> --channel conda-forge
```

Before [ign-rendering](https://github.com/ignitionrobotics/ign-rendering) becomes available on conda-forge, follow its tutorial to build it from source.

#### Build from source

1. Activate the Conda environment created in the prerequisites:

    ```
    conda activate ign-ws
    ```

1. Navigate to where you would like to build the library, and clone the repository.

    ```
    # Optionally, append `-b ign-gui#` (replace # with a number) to check out a specific version
    git clone https://github.com/ignitionrobotics/ign-gui.git
    ```

1. Configure and build

    ```
    cd ign-gui
    mkdir build
    cd build
    ```

>     Before ``ign-rendering`` becomes available on conda-forge, we need to build it from source and specify the path containing ``ignition-rendering<#>-config.cmake`` and ``ignition-rendering<#>-ogre-config.cmake`` in ``CMAKE_PREFIX_PATH``. That path could be ``ign-rendering-install-path\lib\cmake``, for example.

Reviewer: ```suggestion
    As``ign-rendering`` is not available on conda-forge, we need to build it from source and specify the path containing ``ignition-rendering<#>-config.cmake`` and ``ignition-rendering<#>-ogre-config.cmake`` in ``CMAKE_PREFIX_PATH``. That path could be ``ign-rendering-install-path\lib\cmake``, for example.
```
